tests/unit/api/apiUtils/versioning.js

```diff
@@ -43,6 +43,24 @@ describe('versioning helpers', () => {
                     },
                 },
             },
+            {
+                // prior MPU object non-null version exists
```

Comment:
instead of commenting the description of the test case, it might be better to do:
`{ description: 'prior MPU object non-null version exists }`
and incorporate this into `it(\`..${ testCase.description }..\`` in forEach